### PR TITLE
Connorwon 226 mobile view v2 sidebar

### PR DESCRIPTION
**Pull request by ConnorWon**
## Description

Made sidebar stick to top for projectsV2 mobile view

## Updates

added a state to AppRedux which tracks if user is viewing the projectsV2 pages
made nav bar box shadow change depending on if in projectsV2 pages

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [x] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by ConnorWon** [comment]
[mobile view v2 sidebar](https://trello.com/c/7wa29JO3/226-mobile-view-v2-sidebar)

File: src/pages/ProjectV2/Wrapper/ProjectV2Wrapper.css [hunk 1]
```diff
@@ -98,6 +108,48 @@
...
+        a {
+            text-decoration: none;
+            font-size: 20px;
+            font-family: 'Open Sans';
+        }
+    }
+
+    .project-content-container-title {
+        color: black;
+        font-weight: bold;
```
Comment (ryanyae) [inline src/pages/ProjectV2/Wrapper/ProjectV2Wrapper.css:138]: Maybe just a little bit heavier? I don't know the default weight of a bold text but maybe around 700-800?

**Review by ryanyae — CHANGES_REQUESTED** [review]
Great job! I just have the one requested change and could you maybe change the colour for mobile view because the desktop navbar is white but the mobile navbar is like a gray, 
image
.

File: src/pages/ProjectV2/Wrapper/ProjectV2Wrapper.css (hunk 1 above)
Comment (ConnorWon) [reply src/pages/ProjectV2/Wrapper/ProjectV2Wrapper.css]: Done. Had to change the import for Open Sans in App.css

**Review by ElisaHu1 — APPROVED** [review]
this looks awesome!
thank you for your hardwork Connor!

**Review by ryanyae — APPROVED** [review]

**Pull request merged**